public/app/features/plugins/sql/types.ts

```diff
@@ -182,249 +187,25 @@ export interface MetaDefinition {
   kind: CompletionItemKind;
 }
 
-/**
- * Provides a context for suggestions resolver
- * @alpha
- */
+export {
+  CompletionItemKind,
+  LanguageCompletionProvider,
+  LinkedToken,
+  ColumnDefinition,
+  CompletionItemPriority,
+  StatementPlacementProvider,
+  SuggestionKindProvider,
+  TableDefinition,
+  TokenType,
+  OperatorType,
+  StatementPosition,
+} from '@grafana/experimental';
+
+// TODO: export from @grafana/experimental
```

Comment:
Maybe wait for a release from experimental before we merge this so we can remove this todo.

Reply:
updated